js/ui/drop_down_button.js

```diff
@@ -169,7 +169,10 @@ const DropDownButton = Widget.inherit({
         this.$element().addClass(DROP_DOWN_BUTTON_CLASS);
         this._renderButtonGroup();
         this._updateArrowClass();
-        this._loadSelectedItem().done(this._updateActionButton.bind(this));
+
+        if(this.option('selectedItemKey')) {
```

Comment:
It's better to use `isDefined` utility
// selectedItemKey === 0

Reply:
fixed